### PR TITLE
Stress test proxy finished

### PR DESCRIPTION
**Pull request by michaelrtobias**

### CODE REVIEW

**Comment by michaelrtobias** [comment]
@chanceknguyen @bland0

File: client/public/importCss.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+@import 'css/addtocart.css';
+@import 'css/App.css';
+@import 'css/buttonLink.css';
+@import 'css/darkBox.css';
+@import 'css/infoPannel.css';
+@import 'css/ITGRTY.css';
+@import 'css/languages.css';
+@import 'css/pannelBar.css';
+@import 'css/socialPannel.css';
+@import 'css/titleLine.css';
```
Comment (chanceknguyen) [inline client/public/importCss.css:12]: Did the legacy codebase not use CSS in JS or a CSS processor or anything?

**Review by chanceknguyen — APPROVED** [review]
LGTM

File: client/public/importCss.css (hunk 1 above)
Comment (michaelrtobias) [reply client/public/importCss.css:12]: nope. just straight up css and it was all split up by files.

Comment (michaelrtobias) [reply client/public/importCss.css:12]: so this was the easiest way to not screw up the front end

**Pull request merged**